site/content/blogs/release-v126.en.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Can you add the blog to mkdocs.yml as well 🥺

Reply:
i was about to add it once it's finished but im ok for both.